tsl/src/init.c

```diff
@@ -372,15 +245,11 @@ _PG_init(void)
 	 */
 	ts_license_enable_module_loading();
 
-#if PG_VERSION_SUPPORTS_MULTINODE
 	_remote_connection_init();
-#endif
 }
 
 PGDLLEXPORT void
 _PG_fini(void)
 {
-#if PG_VERSION_SUPPORTS_MULTINODE
 	_remote_connection_fini();
```

Comment:
I realize this isn't within the scope of this PR and that module finilization/cleanup isn't really supported, but it is odd to only cleanup connections here. Maybe connection cleanup should move to module_shutdown() and have that shudown function called from here?

(Optional to fix in this PR. Just mental note).